.github/workflows/tests.yml

```diff
@@ -36,10 +42,11 @@ jobs:
         run: |
           pip install setuptools pip --upgrade
           pip install -v -e ".[test]"
+          pip install pytest-cov
       - name: Show python environment
         run: |
           python --version
           python -m pip list
       - name: Run python tests
         run: |
-          pytest . --tb=long -svv
+          pytest . --tb=long -svv --cov=jupyterlab_server
```

Comment:
:+1: are we gonna send coverage anywhere? If nothing else, i'm a fan of:
```
--cov-report term:skip-covered --cov-fail-under XX
```
Where XX is `whatever we have now - 1`

Reply:
We could 🙃

@blink1073 do we use codecov or something?

Reply:
We aren't actively managing coverage in this org iirc but codecov is what I typically use.  I can add integration for this repo tomorrow morning.

Reply:
Thanks!

Reply:
We did have it set up for [`pytest-check-links`](https://github.com/jupyterlab/pytest-check-links/blob/master/.travis.yml)  I added you as a maintainer on this repo @goanpeca and created https://codecov.io/gh/jupyterlab/jupyterlab_server

Reply:
@bollwyvl you already had team access on the repo

Reply:
Ok @blink1073 thanks!